default.nix

```diff
@@ -1,22 +1,14 @@
+args@{ cudaVersion ? null, ...}:
```

Comment:
I'd suggest removing the `...` from this argument list.

Nix doesn't allow you to pass arguments from the command line that aren't directly specified in an argument list.  I've seen at least 5 or 6 people separately run into this limitation (myself included): https://github.com/NixOS/nix/issues/1360

(The only reason you'd want `...` here is if you plan on programatically importing this `default.nix` file from a separate nix derivation.)

Reply:
I think it could make sense for a third party to pass arguments to `default.nix`. If someone wants to import this repo and its nix expressions, then arguments passed here will be routed to `shared.nix` where they can affect a bunch of things. I'll think I'll keep this as is.